conans/client/client_cache.py

```diff
@@ -164,4 +164,6 @@ def load_profile(self, name):
         return Profile.loads(text)
 
     def current_profiles(self):
+        if not os.path.isdir(self.profiles_path):
+            return []
```

Comment:
Might be better to raise a ConanException, so the user knows that the path for profiles (we will also add this functionality in 0.17 release), instead of silently showing there are not profiles